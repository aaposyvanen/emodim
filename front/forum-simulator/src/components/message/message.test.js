import React from 'react';
import renderer from 'react-test-renderer';
import Message from "./message";

describe("message", () => {
    it("renders correctly", () => {
        const component = renderer.create(
            <Message />,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
