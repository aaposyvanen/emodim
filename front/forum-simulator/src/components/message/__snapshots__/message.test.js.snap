// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders correctly with props 1`] = `
<div
  className="message-box"
>
  <div
    className="metadata"
  >
    <div
      className="author"
    >
      Author
    </div>
    <div
      className="date"
    >
      2000-01-01 00:00:00
    </div>
  </div>
  <div
    className="message"
  >
     test
  </div>
</div>
`;

exports[`Message renders correctly without props 1`] = `null`;
